modules/cnf-nrop-support-schedulable-resources.adoc

```diff
@@ -0,0 +1,20 @@
+// Module included in the following assemblies:
+//
+// *scalability_and_performance/cnf-numa-aware-scheduling.adoc
+:_mod-docs-content-type: CONCEPT
+[id="cnf-numa-resource-operator-support-scheduling-cp_{context}"]
+= NUMAResourcesOperator support for schedulable control-plane nodes
+
+Traditionally, control plane nodes in {product-title} are dedicated to running critical cluster services. By enabling schedulable control plane nodes, you allow user-defined pods to be scheduled on these nodes. This effectively turns the nodes into a hybrid of control plane and worker nodes, which is especially useful in resource-constrained environments, such as compact clusters. For example, when you install a three-node cluster, the three control plane machines also act as compute machines. This configuration is part of the installation process.
+
+You can make control plane nodes schedulable by setting the `mastersSchedulable` field to true in the `schedulers.config.openshift.io` resource.
+
+The NUMA Resources Operator ensures topology-aware scheduling for workloads that require specific NUMA affinity. When control plane nodes are made schedulable, the NUMA Resources Operator extends its functionality to manage these nodes. The NUMA Resources Operator ensures that NUMA-aware pods are scheduled on a node with the best NUMA topology, even if that node is a control plane.
+
+When configuring the NUMA Resources Operator, its management scope is determined by the `nodeGroups` field in its custom resource (CR). This principle applies to both compact and multi-node clusters.
+
+Compact clusters:: In a compact cluster, all nodes are configured as schedulable control plane nodes. The NUMA Resources Operator is configured to manage all nodes in the cluster.
```

Comment:
>  The NUMA Resources Operator is configured to manage all nodes in the cluster.

again this is in the hands of the user, you can point instead that "The NUMA Resources Operator can be configured to manage all nodes in the cluster. Follow the deployment instructions for more details on the process".